src/tplr/schemas.py

```diff
@@ -49,3 +45,22 @@ def __eq__(self, other):
     model_config = ConfigDict(
         str_strip_whitespace=True,
     )
+
+
+class CommsGetResult(BaseModel):
+    """A standard return type for the `get` function."""
+
+    data: Optional[Dict[str, Any]] = Field(
+        None, description="The data retrieved by the get function."
+    )
+    global_step: Optional[int] = Field(
+        None, description="The global step associated with the data."
+    )
```

Comment:
Honestly. Not sure if we even need this. But to keep it as a refactor we can keep it. But eventually we can rip out the need to return this. pretty sure it is just discarded anyways all in the code.